docs/visualization/heatmaps.ipynb

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
<p>For eg -&gt; For example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done